deployments/charts/kuma/Chart.yaml

```
@@ -2,8 +2,8 @@ apiVersion: v2
 name: kuma
 description: A Helm chart for the Kuma Control Plane
 type: application
-version: 0.3.1
-appVersion: 0.7.2
+version: 0.3.2
```

```suggestion
version: 0.4.0
```
nit, I think this might make sense as a minor bump for the chart, but totally semantic and not necessary to change

---

I guess I'm just not sure how we want to go about versioning this chart -- if we keep branch-specific versions in just the `major.minor.patch` format, it may get hard to not conflict between branches. If we lock the chart version to the appVersion, we wouldn't be able to add features/ patch bugs without releasing an entirely new version of kuma as well.

---

Semantically you are right. Since we are on the schedule with this release I'd like to use 0.3.2, but let's start a discussion how we want to version charts. I'll create Github issue for it